neos/flow-development-collection
# Update: ignore disabled authentication providers

**What I did**
Ignore null providers in the provider factory

**How I did it**
Ignore null providers in TokenAndProviderFactory::buildProvidersAndTokensFromConfiguration foreach loop

**How to verify it**
Set `Neos.Neos:Backend` provider to `null`, `~` or `false` in order to disable it and check for php errors on next login request

**Checklist**


Fix #2606
